sdk/confidentialledger/confidential-ledger-rest/package.json

```diff
@@ -89,11 +85,11 @@
     "@azure/core-paging": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.1.0",
     "@azure/logger": "^1.0.0",
-    "autorest": "^3.6.1",
```

Comment:
did you mean to add it in the devDep instead?

Reply:
we don't usually depend on autorest in the package, it's an expectation that devs install this globally